src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -18,6 +20,8 @@ public partial class WorkspaceDependencyView : UserControl
 
         private WorkspaceModel currentWorkspace;
 
+        private DynamoViewModel dynamoViewModel;
```

Comment:
don't do this please! let's figure out what we need to add to the extensions API to accomplish this without passing around the entire viewModel.

Reply:
@mjkkirschner We need to access `PackageManagerClientViewModel` to download package here, what do you recommend? Seems we could just expose the download package part to be an API, there are a few examples there, @scottmitchell